fcm_firbase_admin.go

```diff
@@ -15,98 +15,72 @@ func (this *FcmMsg) makeMulticastMessageData() (*map[string]string, bool) {
 	}
 
 	data, ok := this.Data.(map[string]interface{})
+	dataMap := make(map[string]string)
```

Comment:
Adding only those fields to the `dataMap` that we actually get from upstream
For eg not always do we have the `Actions` coming in from bonito and the previous change was still adding the `actions` key to the datamap

This ^ led to crashes in Android because we ran into an NPE [here](https://github.com/fishbrain/android/blob/29c1553898a44963bf5a30c198a584f3723a9acb/FishBrainApp/src/main/java/com/fishbrain/app/presentation/push/PushNotificationViewModel.kt#L31) in Android